### ISSUE TITLE
Class weights and coral.OrdinalCrossEntropy()

### ISSUE THREAD

**Issue by Bsingstad** [ck37/coral-ordinal#26]
Is it possible to use the `class_weight` parameter of the `fit()` function while using `coral.OrdinalCrossEntropy()`?

**Comment by gmgeorg** [comment]
@Bsingstad have you tried?  does it not work / fail?  

The OrdinalCrossEntropy() loss function supports `sample_weight` arguments and implements a loss = loss * sample_weights calculation if provided by user.

**Comment by ck37** [comment]
Closing due to lack of response. Please feel free to re-open or post a new issue if it would be helpful.

Cheers,
Chris
